tests/MalibuSprites_test.js

```diff
@@ -22,26 +19,19 @@ describe('MalibuSprite', () => {
   it('fetches the latest version of the sprites', () => {
     const url = 'https://www.herokucdn.com/malibu/latest/sprite.svg'
     fetchMock.get(url, sprites)
-    mount(<MalibuSprites />)
+    render(<MalibuSprites />)
     expect(fetchMock.lastUrl()).to.equal(url)
   })
 
   it('fetches different sprites when set is provided', () => {
     const set = 'marketing'
     const url = `https://www.herokucdn.com/malibu/latest/${set}/sprite.svg`
     fetchMock.get(url, sprites)
-    mount(<MalibuSprites set={set} />)
+    render(<MalibuSprites set={set} />)
     expect(fetchMock.lastUrl()).to.equal(url)
   })
 
-  it('correctly sets the content of sprites when fetched', () => {
```

Comment:
This test is not feasible with react-testing-library. In addition, with the previous implementation, changing state in the tests just to check that state was changed doesn't make much sense when `setState` for react class components is part of react's core API.

Reply:
Agree, it is puzzling that we seemed to be just testing the `setState` function.